drgadget/ropviewer.py

```diff
@@ -360,17 +353,15 @@ def OnKeydown(self, vkey, shift):
             elif vkey == ord("S"):
                 self.export_binary()
 
-
-        # colon
-        elif vkey == 190:
+        elif vkey == 186:  # colon
```

Comment:
vkey == 190 works on my box, 186 doesn't. Could you please replace line 356 with the following code and let me know whether it works? Thank you.

       elif shift == 1:
            if vkey == 190:
                self.set_comment(self.GetLineNo())